core/EAP7-1266_WFCORE-4314_enchance_keystore_commands_alias.adoc

```diff
@@ -0,0 +1,343 @@
+96:e6:1e:4b:d0:90:4a:bd:ac:8f:37:ee:b1:e0:9f:31:82:d2:3c:90:43:cb:64:2f:88:00:41:60:ed:f9:ca:09:b3:20:76:a7:9c:32:a6:27:f2:47:3e:91:87:9b:a2:c4:e7:44:bd:20:81:54:4c:b5:5b:80:2c:36:8d:1f:a8:3e:d4:89:e9:4e:0f:a0:68:8e:32:42:8a:5c:78:c4:78:c6:8d:0
+5:27:b7:1c:9a:3a:bb:0b:0b:e1:2c:44:68:96:39:e7:d3:ce:74:db:10:1a:65:aa:2b:87:f6:4c:68:26:db:3e:c7:2f:4b:55:99:83:4b:b4:ed:b0:2f:7c:90:e9:a4:96:d3:a5:5d:53:5b:eb:fc:45:d4:f6:19:f6:3f:3d:ed:bb:87:39:25:c2:f2:24:e0:77:31:29:6d:a8:87:ec:1e:47:48:f8
+:7e:fb:5f:de:b7:54:84:31:6b:22:32:de:e5:53:dd:af:02:11:2b:0d:1f:02:da:30:97:32:24:fe:27:ae:da:8b:9d:4b:29:22:d9:ba:8b:e3:9e:d9:e1:03:a6:3c:52:81:0b:c6:88:b7:e2:ed:43:16:e1:ef:17:db:de:03:82:01:05:00:02:82:01:00:7f:3d:31:7e:55:de:22:6b:0a:15:a1:
+f4:c7:aa:8e:a0:7f:ed:bc:dd:38:4d:44:6b:ac:06:bb:a0:21:53:cb:1b:dc:97:3d:3c:38:97:cb:6f:3e:76:07:91:46:21:34:2a:8d:02:52:98:a5:5a:2e:22:12:d6:1a:72:6c:c1:5a:20:c0:7c:11:70:31:5f:09:07:cf:21:e1:4f:19:9f:b0:f7:83:af:10:15:46:46:3b:11:46:63:24:d6:4
+4:be:ce:09:16:d9:4d:dd:49:73:23:5a:2c:12:40:19:12:dd:a7:f8:59:d2:5d:72:a5:9c:ab:42:f9:f6:77:4d:4d:c0:2a:d8:bc:fb:f8:ca:c3:6f:66:3c:28:c2:f4:1e:b7:09:0b:ba:14:a8:09:9d:26:40:40:ba:ba:b5:ce:52:62:8d:dd:41:aa:d9:8f:d8:3f:e7:33:44:3a:40:fa:3a:de:28
+:93:83:e2:03:ac:b1:57:9d:7b:ac:4e:1b:0c:2f:2c:85:f0:64:3c:83:61:c0:47:1b:e5:70:ce:6d:dd:90:4a:28:59:b1:b5:34:54:d3:63:eb:f9:4a:2e:c1:92:d6:94:7d:f7:65:d0:44:57:2b:7a:64:d6:3a:7e:95:67:e0:95:55:b0:e0:0d:be:3a:3a:fb:5d:95:97:42:1c:48:f8:1c:d1:52:
+57:a3:21:30:1f:30:1d:06:03:55:1d:0e:04:16:04:14:77:73:75:9a:5b:1e:d9:e0:24:4a:d1:94:5b:26:0f:0d:ff:d5:0f:f5:30:0b:06:09:60:86:48:01:65:03:04:03:02:03:40:00:30:3d:02:1d:00:94:ba:8e:2f:40:34:f6:fe:9f:30:9e:69:36:58:fa:88:bb:f5:78:67:5a:92:cd:89:3
+e:a9:e1:ba:02:1c:1c:6f:d9:27:db:54:79:2d:76:80:5e:b2:60:0c:60:90:d0:fb:be:cf:03:91:8b:0a:ef:af:58:40",
+                "subject" => "CN=www.example.com",
+                "issuer" => "CN=www.example.com",
+                "not-before" => "2019-05-27T08:29:31.000+0200",
+                "not-after" => "2019-08-25T08:29:31.000+0200",
+                "serial-number" => "f0:2c:f7:ba:47:a2:33:c1",
+                "signature-algorithm" => "SHA256withDSA",
+                "signature" => "30:3d:02:1d:00:94:ba:8e:2f:40:34:f6:fe:9f:30:9e:69:36:58:fa:88:bb:f5:78:67:5a:92:cd:89:3e:a9:e1:ba:02:1c:1c:6f:d9:27:db:54:79:2d:76:80:5e:b2:60:0c:60:90:d0:fb:be:cf:03:91:8b:0a:ef:af:58:40",
+                "version" => "v3"
+            }]
+        }
+    }
+}
+--
+
+
+NOTICE: `read-alias(alias=...)` and `read-alias(alias=..., verbose=true)` difference is the same as for `read-aliases`.
+
+
+== Test Plan
+
+ Modify existing key store tests to verify features don't break anything. Ensure QE can test HAL.
+
+== Community Documentation
+////
+No documentation needed, command help should be enough.
```

Comment:
Documentation on the new parameters should be added to the "Using the Elytron Subsystem" section in the WildFly documentation.